src/nodes/primitive/range.ts

```diff
@@ -26,14 +27,38 @@ export const comparators = {
 
 export type Comparator = keyof typeof comparators
 
-export const comparatorDescriptions = {
+export const numericComparatorDescriptions = {
     "<": "less than",
     ">": "more than",
     "<=": "at most",
     ">=": "at least",
     "==": "exactly"
 } as const satisfies Record<Comparator, string>
 
+export const dateComparatorDescriptions = {
+    "<": "before",
+    ">": "after",
+    "<=": "at or before",
+    ">=": "at or after",
+    "==": "the day"
```

Comment:
We don't want to write "the day" here for similar reason I mentioned about needing to change the description. Just use "at"